index.html

```diff
@@ -0,0 +1,59 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+	<link rel="preconnect" href="https://fonts.gstatic.com">
+	<link href="https://fonts.googleapis.com/css2?family=Raleway:wght@500;600;700;800&display=swap" rel="stylesheet">
+	<link href='https://fonts.googleapis.com/css?family=Roboto' rel='stylesheet'>
+	<link rel="stylesheet" href="style.css" />
+	<title>Excercise2</title>
+</head>
+
+<body>
+	<div class="outer">
+		<div class="dev">
+			<h3 class="heading">Why Loconav's Vehicle Tracking System ?</h3>
+			<div>
+				<div class="row">
+					<div class="column" style="background-color: #FFF"> <img src="image1.png" class="vehicle-track-image">
+						<h4 class="headinternal">INTELLIGENT VEHICLE TRACKING</h4>
+						<p class="description"> Wheather you are single vehicle or a <span style="color: #6399C7">fleet owner</span>, Loconav is determined to protect and monitor your asset from theft or harm</p>
+					</div>
+					<div class="column" style="background-color: #FFF"> <img src="image4.png" class="vehicle-lock-image">
+						<h4 class="headinternal">
+							VEHICLE LOCK</h4>
+						<p class="description"> Turn on vehicle lock from Loconav's <span style="color: #6399C7">mobile app</span> and be assured that your vehicle won't start without your permission</p>
+					</div>
+					<div class="column" style="background-color: #FFF"> <img src="image2.png" class="live-alert-image">
+						<h4 class="headinternal">
+							LIVE ALERTS</h4>
+						<p class="description"> Get live over-speeding alerts,alerts on entry and exit points,monitor idling in real time ,etc. Vehiccle services and maintenance alerts</p>
+					</div>
+					<div class="column" style="background-color: #FFF"> <img src="image3.png" class="vehicle-health-image">
+						<h4 class="headinternal">
+							VEHICLE HEALTH</h4>
+						<p class="description"> Get diagostic,mileage and health report of your vehicle so that you can worry less about your vehicle's well-being and enjoy your time on road</p>
+					</div>
+				</div>
+			</div>
+		</div>
+		<div class="loginform">
+			<h3 class="tagline"> Get an offer on your mobile</h3>
+			<input type="text" name="phonenumber" placeholder="Phone Number" class="inputd">
+			<input type="number" name="noofvehicle" placeholder="How many vehicles ?" class="inputd">

```

Comment:
please look up min property for the input tag. We should not be able to set this to negative values.